rpc/tests/integration.rs

```diff
@@ -224,6 +224,16 @@ mod endpoints {
         assert_eq!(header.hash(), block_id.hash);
     }
 
+    #[test]
+    fn commit_height_1() {
```

Comment:
Nice to have a case, when I tried it out I went straight into the `lite_impl` module. As we likely gonna fix the `fn hash` implementation, shouldn't we test deeper in the stack?

Reply:
Probably both?